intent_config_test.go

```diff
@@ -411,6 +425,7 @@ func TestV3IntentConfigWalletDeployment(t *testing.T) {
 	// This is needed since the `CreateIntentBundle` function expects an array of `CallsPayload`.
 	mockPayloadInitialBatch := []v3.CallsPayload{mockPayloadCall}
 	mockBundles := []v3.CallsPayload{}
+	metaTxnIds := []string{}
```

Comment:
btw, in v3 we call them just "operations" :D instead of "meta transactions".. I am pretty sure, but we can check with Agustin..